high_card.rb

```diff
@@ -1,5 +1,24 @@
 # Use these two arrays to generate a deck of cards.
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
-
 players = []
+
+deck = []
+# game = []
+deck = ranks.product(suits)
+ puts deck.shuffle
```

Comment:
Note that this won't permanently shuffle the deck... it'll only display a temporarily shuffled deck... if you look at deck again on the next line, you'll see that it's back to it's original order.